src/Sil/Bundle/UserBundle/Entity/Traits/Nameable.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licenced under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Sil\Bundle\UserBundle\Entity\Traits;
+
+/* @todo: remove this or not or use this */
+
+trait Nameable
```

Comment:
Use Nameable trait from blastBaseEntitiesBundle instead

Reply:
Not the same usage, as here is for firstname and lastname, the one from blast is from for name only